workshops/py_Christmas/README.md

```diff
@@ -0,0 +1,380 @@
+
+Last function in this file to make the tiny balls on either sides of the boxes (green part) of the Christmas tree.
+Go through the code. You will probably understand it pretty easily.
+```python
+def balls(turtle, color, x, y, radius):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x, y)
+    turtle.pendown()
+    turtle.begin_fill()
+    turtle.circle(radius)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+
+``` 
+Yes, ******turtle****** library has a circle function. Made our job pretty easy, didn't it? 
+
+ Our functions to make the figures is ready. Let's take a look at the whole code.
+
+```python
+import turtle
+def box1 (turtle, color, x, y, width, height):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x,y)
+    turtle.pendown()
+    turtle.begin_fill()
+    for i in range(2):
+        turtle.forward(width)
+        turtle.left(90)
+        turtle.forward(height)
+        turtle.left(90)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+
+
+
+def star(turtle, color, x, y, size):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x, y)
+    turtle.pendown()
+    turtle.begin_fill()
+    for i in range (5):
+        turtle.forward(size)
+        turtle.right(144)
+        turtle.forward(size)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+
+
+def balls(turtle, color, x, y, radius):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x, y)
+    turtle.pendown()
+    turtle.begin_fill()
+    turtle.circle(radius)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+``` 
+All good? 
+
+# Now, lets move on to the next file : main.py
+
+
+Now open the main.py file. 
+
+In this page, we are going to make a brown rectangle which will serve as the tree stem. Then, we are going to make multiple green rectangles stacked above each other. If each rectangle is smaller than the one below it, the whole image will have a Christmas tree like appearance.
+Shall we begin?
+
+```python
+from turtle import *
+from Figures import *
+from random import randint
+
+```
+Here, we import all fuctions from 'turtle' library, all functions from 'Figures' file (which we just created) and finally 'randint' function from 'random' library so that we can use it to generate a random integer.
+Why do we need a random integer? We'll see .
+
+
+```python
+speed(0)
+
+bg = turtle.Screen()
+bg.bgcolor("#99004D") #Setting back ground color
+
+y = -100
+width = 240
+
+```
+`speed()` function determines the speed of the turtle, ie, the speed in which the figures get drawn on the screen. When we set that to zero it moves as fast as it can. 
+Next, we set the background color to maroon using hexcode (#99004D).
```

Comment:
Add another blank line before this. Example:

`speed()` function determines the speed of the turtle, ie, the speed in which the figures get drawn on the screen. When we set that to zero it moves as fast as it can.

Next, we set the background color to maroon using hexcode (#99004d).